platform/ios/PolyPodApp/PolyPodUITests/SmokeTest.swift

```diff
@@ -0,0 +1,93 @@
+// Please remove this line and the empty one after it
+
+import XCTest
+
+class SmokeTest: XCTestCase {
+    private let app = XCUIApplication()
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+    }
+
+    func testOpenFeaturesNoErrorPopUp() throws {
+        app.launchWithArgs()
+        assertHomeScreenIsShown()
+        
+        let tile_identifiers = ["big_card_view", "medium_card_view", "small_card_view"]
+        
+        for identifier in tile_identifiers {
+            var count = 0
+            var exists = true
+            while exists {
+                let tile = featureTileView(index: count, indentifier: identifier)
+                exists = tile.exists
+                count += 1
+                if exists {
+                    print("\(identifier) - \(count)")
+                    openFeatureAndCheckForErrorPopup(tile: tile)
+                }
+            }
+        }
+    }
+
+    private func assertHomeScreenIsShown() {
+        XCTAssertTrue(
+            homeScreenView().exists,
+            "Onboarding was supposed to be shown"
+        )
+    }
+
+    private func homeScreenView() -> XCUIElement {
+        app.otherElements["homescreen_view"]
+    }
+    
+    private func featureView() -> XCUIElement {
+        app.otherElements["feature_view"]
+    }
+
+    private func featureCloseButton() -> XCUIElement {
+        app.otherElements["feature_close_button"]
+    }
+    
+    private func featureErrorPopup() -> XCUIElement {
+        app.alerts["feature_error_popup"]
+    }
+
+    private func featureTitleText() -> XCUIElement {
+        app.otherElements["feature_title_text"]
+    }
+
+    private func featureTileView(index: Int, indentifier: String) -> XCUIElement {
+        app.otherElements.matching(identifier: indentifier).element(boundBy: index)
+    }
+    
+    private func openFeatureAndCheckForErrorPopup(tile: XCUIElement) {
+        var featureViewVisible = false
+        var swipeDownCount = 0
+        
+        while !featureViewVisible {
+            tile.tap()
+            featureViewVisible = featureView().waitForExistence(timeout: 1)
+            if !featureViewVisible {
+                // Need to swipe up so that the tile becomes visible and the tap succeeds
+                homeScreenView().swipeUp()
```

Comment:
Interesting, in this PR https://github.com/polypoly-eu/polyPod/pull/1144/files, after doing some tests, the UI tests did automatically perform the scroll. It fails to do so in this case?

Reply:
Here's what I get in the logs without something like that:

```
 t =     4.99s     Check for interrupting elements affecting "test_feature" Other
    t =     5.03s     Synthesize event
    t =     5.05s         Scroll element to visible
    t =     5.06s         Find the "test_feature" Other
    t =     5.12s         Computed hit point {196, 504.75} after scrolling to visible
```

What's even more odd to me: We only get here for `tile` objects that have `exists == true` - it exists, but cannot be tapped?

Reply:
There are tiles that can be tapped without having to scroll. And for some of them I need to scroll in order to tap them. 

Interestingly, it might have something to do with the size of the tile. Full screen width tiles don't have this issue.